db/migrate/20200107034249_add_dispatched_quantity_to_orders_packages.rb

```diff
@@ -0,0 +1,5 @@
+class AddDispatchedQuantityToOrdersPackages < ActiveRecord::Migration
+  def change
+    add_column :orders_packages, :dispatched_quantity, :integer, default: 0
```

Comment:
Can also add the following instead of the rake task. See https://guides.rubyonrails.org/active_record_migrations.html#using-reversible

```
    reversible do |dir|
      dir.up do
        # initialise the dispatched_quantity values
        execute <<-SQL
          UPDATE orders_packages SET dispatched_quantity=quantity;
        SQL
      end
    end
```